### PR TITLE
Catch Interruptions, Automation and Refactor

### PR DESCRIPTION
**Pull request by fernandovalera**
Refactored run-tests.sh and added catch to interruption so that python alarm script is killed if the script exits early.

### CODE REVIEW

**Comment by fernandovalera** [comment]
Update includes full automation for load balancing setup and teardown. Scripts are documented and the README contains help for running all tests or just spinning up system to test individually.

File: run-tests.sh [hunk 1]
```diff
@@ -1,65 +1,82 @@
...
 
-
+alarm_monitor_script_pid=
+
+# Capture "ctrl + c" interruptions and kill the alarm monitor script if it is running
+exit_fn () {
+    trap SIGINT             # Restore signal handling for SIGINT
+    if ! [[ "alarm_monitor_script_pid" = "" ]]
+    then
+        kill ${alarm_monitor_script_pid}
```
Comment (javenschuetz) [inline run-tests.sh:10]: do we need sudo?

File: shutdown_load_balancing.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    aws autoscaling detach-load-balancer-target-groups --auto-scaling-group-name PicSiteASG \
+        --target-group-arns ${asg_target_group_arns}
+
+    if [[ "${?}" = "0" ]]
+    then
+        echo "Detached target groups with ARNs \"${asg_target_group_arns}\""
+    fi
+else
+    echo "No target group found!"
+fi
```
Comment (javenschuetz) [inline shutdown_load_balancing.sh:50]: blank lines are nice

**Review by javenschuetz — APPROVED** [review]
looks good

**Pull request merged**